.github/workflows/generator_container_slsa3.yml

```diff
@@ -82,6 +101,8 @@ jobs:
       packages: write # Needed to login and upload attestations to ghcr.io.
     steps:
       - name: Generate builder
+        id: generate-builder
+        continue-on-error: true
```

Comment:
I wonder if we can pass the `continue-on-error` input here?
https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability

Also on `cosign-install`, `login`, and `sign-prov`. I assume `login` and `sign-prov` should work at least since those are just basic job steps.

Reply:
no working because it's interpreted as a string. `${{ inputs.continue-on-error == 'true' }}` also interpreted as string. See https://github.com/laurentsimon/slsa-on-github-test/blob/main/.github/workflows/run-error.yml#L6

Reply:
huh. Strange... It's not that important so we could clean it up later if we find a way.